### PR TITLE
Disable Chaingun for bots by default

### PR DESCRIPTION
**Pull request by Gireen**
part for #3055
not tested but should work 

it was mentioned multiple times, and recently again on irc that bots with chaingun are to strong or could be considered a cheat.

until they can be weakened in other ways the chaingun should be disabled for bots by default.

server owners or offline players can enable it on their own anytime if they want to

### CODE REVIEW

**Comment by slipher** [comment]
#3057 is a proposed solution for fixing the "cheating" aspect of not being affected by chaingun view shaking.

**Comment by illwieckz** [comment]
I guess it's good to merge the disablement, even if things like #3057 are coming. People would still be able to re-enable the chaingun and propose a re-enablement later.

**Review by illwieckz — APPROVED** [review]
LGTM.

**Comment by slipher** [comment]
Why? I don't see any really compelling reason to disable it, with or without the view shaking fix.

**Comment by illwieckz** [comment]
Players have bad experience when it is enabled, and leave the game.

**Comment by cu-kai** [comment]
I should add that the primarily played server (Map & Bot Testing) permits players on the team to decide the value of `g_bot_chain` so this PR is unlikely to make the slightest difference, as the `botequip` command there isn't going away as far as I know.

**Comment by DolceTriade** [comment]
Yeah I agree that this "intermediate" solution is not really doing anything.

**Comment by Gireen** [comment]
#3057 sounds like it could improve the situation, how effective it is must be tested.

The worst case scenario, that this PR addresses is a single new player being alone with bots.
Human bots pick relative early in the game lightarmor and chaingun which is for a inexperienced alien an near impossible enemy, even worse when humans appear together with this equipment.

Modded servers, PvE or PvP with many players are not the focus of this change. Unmodified servers and most important Offline play are the places where new players try the game alone and are affected by the bots the most.

Since Offline play can only be updated by a release, this should be addressed before the next one or the situation stays the same for months or even years.

**Comment by necessarily-equal** [comment]
I'm ok with merging this. This seems like a good idea.

**Comment by illwieckz** [comment]
#3057 still being a draft is a good reason why I suggested to disable first, think about it later (maybe re-enable it later).

If I'm right both unvanquished.net and my server have it disabled from config file.

**Comment by VReaperV** [comment]
If this is already done by server hosts who want it to be disabled, I see no value in this pr.

**Comment by illwieckz** [comment]
The idea is to have a default gameplay that works.

**Comment by Gireen** [comment]
Yeah, the default values are what new players are most likely to encounter. If the release cycle is really getting shorter now, it's not quite as urgent anymore and the jitter PR could be tested in the next release. 

So I would suggest that everyone now vote with their thumbs on the PR as with the motionblur and if no jitter PR or any other improvement attempt has been merged before the next release, then this PR will be merged if the vote is positive.

**Comment by Gireen** [comment]
jitter was merged, the vote is a draw or not interesting and releases are more frequent. 
So this PR has become less urgent with more chances to try other things.

**Pull request closed without merging**
